build.rs

```diff
@@ -0,0 +1,105 @@
+use chrono::Utc;
+use std::fs::{File, OpenOptions};
+use std::io::Write;
+use std::path::Path;
+use std::process::Command;
+use std::str;
+
+const BUILD_OUT_FILE: &str = "constants.rs";
+const SOURCE_DIR: &str = "./src";
+
+/*
+Build script (https://doc.rust-lang.org/cargo/reference/build-scripts.html)
```

Comment:
Let's change the docstring according to this https://doc.rust-lang.org/stable/rust-by-example/meta/doc.html